test/test_word_to_markdown.rb

```diff
@@ -29,8 +29,18 @@ class TestWordToMarkdown < Minitest::Test
   end
 
   should 'parse tables' do
-    skip 'Figure out why there are so many new lines'
-    validate_fixture 'table', "| **Foo** | **Bar** |\n| --- | --- |\n| One | Two |\n| Three | Four |"
+    validate_fixture 'table', "| **Foo**| **Bar**|\n| --- | --- |\n| One | Two |\n| Three | Four |"
+  end
+
+  should 'parse tables with newlines' do
+    expected = <<-DOC.chomp
+| **Foo**| **Bar**|
+| --- | --- |
+| One | Two |
+| Three | Four |
+| Five<br>Six<br>Seven | Row<br><br>With<br><br> Newlines |
```

Comment:
It's unclear why the second column has an extra set of `<br>` tags, the columns appear identical in the document.